lib/msal-node/package.json

```diff
@@ -65,12 +66,13 @@
   "module": "dist/msal-node.esm.js",
   "devDependencies": {
     "@types/jest": "^25.1.2",
+    "@types/uuid": "^7.0.0",
     "husky": "^4.2.3",
     "tsdx": "^0.12.3",
     "tslib": "^1.10.0",
     "typescript": "^3.7.5"
   },
   "dependencies": {
-    "@types/uuid": "^7.0.0"
+    "@azure/msal-common": "1.0.0-alpha.0"
```

Comment:
`axios` needs to also be listed here.

Reply:
Yep, just added and committed.

Reply:
tests are failing now. Will fix and update.